### PR TITLE
Add pop-up list of references.

### PR DESCRIPTION
**Pull request by dbrant**

### CODE REVIEW

File: app/src/main/java/org/wikipedia/bridge/JavaScriptActionHandler.kt [hunk 1]
```diff
@@ -38,6 +38,12 @@ object JavaScriptActionHandler {
         return "window.scrollTo(0, document.getElementById('pcs-footer-container-menu').offsetTop - ${DimenUtil.getNavigationBarHeight(context)});"
     }
 
+    @JvmStatic
+    fun scrollToAnchor(anchorLink: String): String {
+        val anchor = if (anchorLink.contains("#")) anchorLink.substring(anchorLink.indexOf("#") + 1) else anchorLink
+        return "var el = document.getElementById('$anchor'); window.scrollTo(0, el.offsetTop - (screen.height / 2)); setTimeout(function(){ el.style.backgroundColor='#ee0'; setTimeout(function(){ el.style.backgroundColor=null; }, 500); }, 250);"
```
Comment (cooltey) [inline app/src/main/java/org/wikipedia/bridge/JavaScriptActionHandler.kt]: Would it be better if it has multiple lines to increase the readability?

File: app/src/main/java/org/wikipedia/page/references/ReferenceListDialog.kt [hunk 2]
```diff
@@ -0,0 +1,116 @@
...
+    }
+
+    override fun onStart() {
+        super.onStart()
+        BottomSheetBehavior.from(view!!.parent as View).peekHeight = DimenUtil
+                .roundedDpToPx(DimenUtil.getDimension(R.dimen.readingListSheetPeekHeight))
+    }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        L10nUtil.setConditionalLayoutDirection(view, callback()!!.linkHandler.getWikiSite().languageCode())
```
Comment (cooltey) [inline app/src/main/java/org/wikipedia/page/references/ReferenceListDialog.kt]: `getWikiSite()` can be changed to `wikiSite`

Comment (dbrant) [reply app/src/main/java/org/wikipedia/page/references/ReferenceListDialog.kt]: ok!

File: app/src/main/java/org/wikipedia/bridge/JavaScriptActionHandler.kt (hunk 1 above)
Comment (dbrant) [reply app/src/main/java/org/wikipedia/bridge/JavaScriptActionHandler.kt]: ok!

**Pull request merged**